Source/Common/Output_Xml.cpp

```diff
@@ -332,6 +382,28 @@ return_value Output_Xml(ostream& Out, std::vector<file*>& PerFile, ostream* Err)
                 }
 
                 // Captions
+                bool Caption_Begin = !Captions_Partial[1].empty() && FrameNumber == Captions_Partial[1].front();
+                bool Caption_End = !Captions_Partial[0].empty() && FrameNumber == Captions_Partial[0].front();
+                if (Caption_Begin && Caption_End)
+                {
+                    Text += " caption=\"single\"";
```

Comment:
As discussed, rather than start/single/end, let's switch to on/off. So @caption="on" implies that that frame has captions and subsequent frames until the end or the next frame/@caption="off" or the end of the <frames>.

Reply:
> switch to on/off.

PR updated accordingly, please review again.